tests/flytekit/unit/extras/tensorflow/records/test_native.py

```diff
@@ -0,0 +1,54 @@
+from typing import Annotated
```

Comment:
Could you change the directory name from `records` to `record` and move `test_native.py` to `test_record.py`?